app/src/androidTest/kotlin/e2e/pageObjects/login/EnvironmentSettingsPage.kt

```diff
@@ -0,0 +1,21 @@
+package e2e.pageObjects.login
+
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import com.x8bit.bitwarden.e2e.pages.LoginPage
+import e2e.pageObjects.Page
+
+class EnvironmentSettingsPage(composeTestRule: ComposeTestRule) : Page(composeTestRule) {
+
+    private val serverURLField by lazy { getElement("ServerUrlEntry") }
```

Comment:
⛏️ We treat Acronyms as whole words when it comes to var names.
```suggestion
    private val serverUrlField by lazy { getElement("ServerUrlEntry") }
```